### ISSUE TITLE
Add MNIST dataset to the dataset-registry for the new example-get-started

### ISSUE THREAD

**Issue by iesahin** [iterative/example-repos-dev#27]
I started to write the experiments tutorial using the original MNIST from [deepai.org](https://deepai.org/dataset/mnist):

Currently I'm using a Google Storage bucket as a remote:

https://console.cloud.google.com/storage/browser/dvc-example-data

After `prepare` and `preprocess`, the `data` dir is like: 

```
[I] ➜ tree data
data
├── prepared
│   ├── mnist-test.npz
│   └── mnist-train.npz
├── preprocessed
│   ├── mnist-test.npz
│   └── mnist-train.npz
├── raw
│   ├── t10k-images-idx3-ubyte.gz
│   ├── t10k-labels-idx1-ubyte.gz
│   ├── train-images-idx3-ubyte.gz
│   └── train-labels-idx1-ubyte.gz
└── raw.dvc
```

- There are 4 raw data files. These are better served with `dvc import-url` or `dvc get` instead of using a common remote, I think. Also instead of adding them one by one, I decided to add `raw/` directory to DVC. 

- `prepared/` directory contains mixed, and shuffled and remixed data, `preprocess/` contains normalized and (optionally) noise-added data. These are all steps in the pipeline. I plan to use tracking directories instead of individual files in these as well. 

- There may be multiple models (one MLP, one CNN, another deep CNN ... ) that can be selected in `params.yaml` by `train.py`. I think it overcomplicates the pipeline. (Which files will the `train.py` depend?) However it can show the experimentation features more clearly. "We select this model with this amount of salt and pepper noise and it gives us this result."

- I plan to write individual model files for different models. Some of them may not run on Katacoda, but overall there will more parameters for the users to try. 

In summary, I need some way to put the data files to a public place. I can use the above URL or you may want to keep the data at the current one: Which way would you recommend? 

@shcheklein @dberenbaum

This is related with iterative/dvc.org#1400 iterative/katacoda-scenarios#60 and probably many others :)

**Comment by iesahin** [comment]
A preliminary `params.yaml` looks like this: 

```yaml
prepare:
  seed: 20210428
  remix: True
  remix_split: 0.20

preprocess:
  seed: 20210428
  normalize: True
  shuffle: False
  add_noise: False
  noise_amount: 0.0004
  noise_s_vs_p: 0.5

train:
  seed: 20210428
  validation_split: 0.2
  model: "mlp"
  optimizer: "Adam"
  loss: "SparseCategoricalCrossentropy" 
  epochs: 10
  batch_size: 128

model_mlp:
  units: 128
  activation: "relu"

model_cnn:
  dense_units: 128
  conv_kernel_size: 3
  conv_units: 32
  dropout: 0.5
```

There may be some others, I think the more parameters we have, the more freedom to experiment we can have while writing the documentation.

**Comment by shcheklein** [comment]
@iesahin let's add raw data to the existing data registry.

let's push processed, models etc to the public (read) remote on S3, similar to the one we use for the `example-get-started`. I can five you access to S3 to push data there.

**Comment by shcheklein** [comment]
> A preliminary params.yaml looks like this:

is it related to this ticket?

**Comment by iesahin** [comment]
> is it related to this ticket?

Ah, right, moved to the other. Thanks. 😄

**Comment by iesahin** [comment]
This is closed by iterative/dataset-registry#7
